wordpress-mobile/WordPress-iOS
# Fix Gutenberg empty fields with on RTL layout

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/14477

gutenberg PR: https://github.com/WordPress/gutenberg/pull/24510
gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2544

![RTL](https://user-images.githubusercontent.com/9772967/89987364-403dbf80-dc7e-11ea-9e12-c0efa5f9331c.png)

To test:
- Please check the steps on https://github.com/WordPress/gutenberg/pull/24510

PR submission checklist:
